scripts/pr2/execute_pr2.py

```diff
@@ -18,20 +18,24 @@
     parser.add_argument("-t", type=float, default=300, help="time out")
     parser.add_argument("--force", action="store_true", help="disable dry option")
     parser.add_argument("--roseus", action="store_true", help="command via roseus")
+    parser.add_argument("--terminate", action="store_true", help="use terminate flag")
+    parser.add_argument("-tf", type=float, default=0.8, help="threthold of terminate flag")
 
     args = parser.parse_args()
     project_name: Optional[str] = args.pn
     time_out: float = args.t
     force: bool = args.force
     roseus: bool = args.roseus
+    use_terminate: bool = args.terminate
+    tf_thre: float = args.tf
```

Comment:
tf_thre => threshold_threshold 
If name is too long it's always better to name it expressively